src/Symfony/Component/Routing/Tests/Fixtures/TestObjectRouteLoader.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Tests\Fixtures;
+
+use Symfony\Component\Routing\Loader\ObjectRouteLoader;
+
+class TestObjectRouteLoader extends ObjectRouteLoader
```

Comment:
I had to extract this test class to a separate file to avoid the "Remaining self deprecation notices".

Reply:
which is legit :)

Reply:
👌 I just discovered that